releasenotes/notes/0.46/prepare-0.46.0-c3857463a3c052f7.yaml

```diff
@@ -0,0 +1,620 @@
+    :class:`.VF2Layout` and :class:`.VF2PostLayout` which will set a layout
+    based on the reported noise characteristics of a backend.
+  - |
+    The :class:`.CrosstalkAdaptiveSchedule` transpiler pass has been deprecated
+    and will be removed in the 1.0.0 release. This pass was not usable any
+    longer because its internal operation was dependent on custom properties
+    being set in the :class:`.BackendProperties` payload of a
+    :class:`.BackendV1` instance. As no backends are setting these fields
+    the pass has been deprecated.
+  - |
+    The :mod:`qiskit.visualization.qcstyle` module is now deprecated and will be removed
+    in the Qiskit 1.0.0 release. Instead you should use
+    :mod:`qiskit.visualization.circuit.qcstyle` as direct replacement.
+
+  - |
+    Deprecated injecting circuit gate operation into the pulse builder context.
+
+    Deprecated
+    :func:`~qiskit.pulse.builder.call_gate`,
+    :func:`~qiskit.pulse.builder.cx`,
+    :func:`~qiskit.pulse.builder.u1`,
+    :func:`~qiskit.pulse.builder.u2`,
+    :func:`~qiskit.pulse.builder.u3`,
+    :func:`~qiskit.pulse.builder.x`
+
+    If you still wish to inject backend calibrated schedules,
+    you can use following pattern instead of calling gate commands.
+
+    .. code-block:: python
+
+      from qiskit.providers.fake_provider import FakePerth
+      from qiskit import pulse
+
+      backend = FakePerth()
+      sched = backend.target['x'][(qubit,)].calibration
+
+      with pulse.build() as only_pulse_scheds:
+        pulse.call(sched)
+
+    Similarly, :class:`.QuantumCircuit` can be injected in the builder context
+    by manually transpiling and scheduling the object.
+
+    .. code-block:: python
+
+      from math import pi
+      from qiskit.compiler import schedule, transpile
+      qc = QuatumCircuit(2)
+      qc.rz(pi/2, 0)
+      qc.sx(0)
+      qc.rz(pi/2, 0)
+      qc.cx(0, 1)
+
+      qc_t = transpile(qc, backend)
+      sched = schedule(qc_t, backend)
+      with pulse.build() as only_pulse_scheds:
+          pulse.call(sched)
+
+    In any case we now recommend to write a minimum pulse program
+    with the builder and attach it to :class:`.QuantumCircuit` via the
+    :meth:`.QuantumCircuit.add_calibration` method as a microcode of a
+    gate instruction, rather than writing the entire program with the pulse model.
+
+    Deprecated
+    :func:`~qiskit.pulse.builder.active_transpiler_settings`
+    :func:`~qiskit.pulse.builder.active_circuit_scheduler_settings`
+    :func:`~qiskit.pulse.builder.transpiler_settings`
+    :func:`~qiskit.pulse.builder.circuit_scheduler_settings`
+
+    Modified module doc of :mod:`qiskit.pulse.builder` related with example
+    code with circuit elements.
+
+    Deprecated arguments :code:`default_transpiler_settings`, :code:`default_circuit_scheduler_settings`
+    in :func:`~qiskit.pulse.builder.build`
+
+    Deprecated argument of :class:`.QuantumCircuit` type in :func:`~qiskit.pulse.builder.call`
+
```

Comment:
The formatting might be weird after applying this suggestion (inline editing is questionable...), I reordered the information as best as I could, removed the note about modifying the docs, and added a bit of product placement for the generic fake backend :D

```suggestion
    Injecting circuit gate operations into the pulse builder context is now deprecated. The deprecation
        affects the following functions:

    * :func:`~qiskit.pulse.builder.call_gate`,
    * :func:`~qiskit.pulse.builder.cx`,
    * :func:`~qiskit.pulse.builder.u1`,
    * :func:`~qiskit.pulse.builder.u2`,
    * :func:`~qiskit.pulse.builder.u3`,
    * :func:`~qiskit.pulse.builder.x`
        
    As well as input arguments of type :class:`.QuantumCircuit` type in :func:`~qiskit.pulse.builder.call`.
    
    If you still wish to inject backend calibrated schedules,
    you can use following pattern instead of calling gate commands.

    .. code-block:: python

      from qiskit.providers.fake_provider import GenericBackendV2
      from qiskit import pulse

      backend = GenericBackendV2(num_qubits=5)
      sched = backend.target['x'][(qubit,)].calibration

      with pulse.build() as only_pulse_scheds:
        pulse.call(sched)

    Similarly, :class:`.QuantumCircuit` can be injected in the builder context
    by manually transpiling and scheduling the object.

    .. code-block:: python

      from math import pi
      from qiskit.compiler import schedule, transpile
      qc = QuatumCircuit(2)
      qc.rz(pi/2, 0)
      qc.sx(0)
      qc.rz(pi/2, 0)
      qc.cx(0, 1)

      qc_t = transpile(qc, backend)
      sched = schedule(qc_t, backend)
      with pulse.build() as only_pulse_scheds:
          pulse.call(sched)

    In any case we now recommend to write a minimum pulse program
    with the builder and attach it to :class:`.QuantumCircuit` via the
    :meth:`.QuantumCircuit.add_calibration` method as a microcode of a
    gate instruction, rather than writing the entire program with the pulse model.

    The following arguments in :func:`~qiskit.pulse.builder.build` have also been deprecated:
 
    * :code:`default_transpiler_settings`
    * :code:`default_circuit_scheduler_settings`
    
    Together with the functions:
   
    * :func:`~qiskit.pulse.builder.active_transpiler_settings`
    * :func:`~qiskit.pulse.builder.active_circuit_scheduler_settings`
    * :func:`~qiskit.pulse.builder.transpiler_settings`
    * :func:`~qiskit.pulse.builder.circuit_scheduler_settings`
```